daily.yml

```diff
@@ -0,0 +1,29 @@
+
+name: Deploy to Unstable
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  flat-manager:
+    name: "Flat Manager"
+    runs-on: ubuntu-latest
+    container:
+      image: docker.io/bilelmoussaoui/flatpak-github-actions
+      options: --privileged
+    steps:
+    - uses: actions/checkout@v2
+    - uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@flat-manager
+      with:
+        bundle: "calculator.flatpak"
+        manifest-path: "io.elementary.calculator.yml"
+        run-tests: "true"
+        repository-name: "elementary"
+        repository-url: "https://flatpak.elementary.io/elementary.flatpakrepo"
+        cache-key: "flatpak-builder-${{ github.sha }}"
+    - uses: bilelmoussaoui/flatpak-github-actions/flat-manager@master
+      with:
```

Comment:
Could add a condition here before `with`
```
        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
```

Reply:
This file will only ever run when a push to master happens, so the if statement is unneeded here